tests/i2c_master_test/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+
+# Get JSON lists
+file(READ ${CMAKE_CURRENT_LIST_DIR}/test_params.json params_json)
+
+# Get individual fields from params_json
+string(JSON speed_list GET ${params_json} SPEEDS)
+string(JSON stop_list GET ${params_json} STOPS)
+string(JSON dir_list GET ${params_json} DIRECTION)
+
+string(JSON speed_list_len LENGTH ${speed_list})
+string(JSON stop_list_len LENGTH ${stop_list})
+string(JSON dir_list_len LENGTH ${dir_list})
+
+# Subtract one off each of the lengths because RANGE includes last element
+
+math(EXPR speed_list_len "${speed_list_len} - 1")
+math(EXPR stop_list_len "${stop_list_len} - 1")
+math(EXPR dir_list_len "${dir_list_len} - 1")
+
+
+set(APP_PCA_ENABLE ON)
+set(XMOS_SANDBOX_DIR    ${CMAKE_CURRENT_LIST_DIR}/../../..)
+include(${CMAKE_CURRENT_LIST_DIR}/../../deps.cmake)
+
+set(HW_TARGETS xs2 xcoreai)
```

Comment:
This is pedantic, since it's a test, but we're mixing up architectures and device names here. So should be xs2 and xs2 or xcore-200 and xcore.ai

Reply:
done! Changed to xs3